### PR TITLE
Add Dockerfile, remove package-lock to avoid issues with stale modules

### PR DESCRIPTION
**Pull request by tomhosking**
Added example Dockerfile. Also removed `package-lock.json` as this should be generated when `npm install` is run.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+# Blob Storage Emulator
+EXPOSE 10000
+# Azure Queue Storage Emulator
+EXPOSE 10001
+# Azure Table Storage Emulator
+EXPOSE 10002
+
+
+CMD ["azurite","-l", "/azurite-data", "--blobHost", "0.0.0.0"]
```
Comment (XiaoningLiu) [inline Dockerfile]: Add a space between `"azurite",` and `"-l"`

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,22 @@
+FROM node:10.6-alpine
+
+WORKDIR /opt/azurite
+
+COPY package.json package.json
```
Comment (XiaoningLiu) [inline Dockerfile]: Should include `package.json.lock` before `npm install`

**Comment by XiaoningLiu** [comment]
Please resolve the conflict and comments before we can merge the PR.

File: Dockerfile (hunk 2 above)
Comment (tomhosking) [reply Dockerfile]: Added command to copy `package-lock.json` to container (and readded it to repo)

File: Dockerfile (hunk 1 above)
Comment (tomhosking) [reply Dockerfile]: Done

**Pull request merged**